app/scripts/lib/channels/duplex.js

```diff
@@ -153,8 +166,11 @@ define(function (require, exports, module) {
      * @param {Object} [data]
      * @return {String}
      */
+    _messageCount: 0,
```

Comment:
Is `_messageCount` a left-over older version of `messageIdSuffix` or something? I can't see it being used...